notebooks/official/pytorch/pytorch-text-sentiment-classification-custom-train-deploy.ipynb

```diff
@@ -0,0 +1,1979 @@
+{
```

Comment:
<p>invoke the endpoint with the deployed model -&gt; invoke the endpoint where the model is deployed</p><p> </p><p>Vertex AI SDK -&gt; the Vertex AI SDK</p><p> </p><p>Torchserve -&gt; TorchServe</p><p> </p><p>API which -&gt; API, which</p><p> </p><p>also Vertex AI -&gt; also a Vertex AI</p><p> </p><p>as below -&gt; as follows</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='88'/>

Reply:
updated.